### PR TITLE
Invalid switch and checkbox

### PR DESCRIPTION
**Pull request by Kiarokh**
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/switch/switch.tsx [hunk 1]
```diff
@@ -115,7 +121,7 @@ export class Switch {
 
     public render() {
         return (
-            <Host>
+            <Host invalid={this.invalid}>
```
Comment (jgroth) [inline src/components/switch/switch.tsx:124]: Why is this needed? When the consumer set the `invalid` prop it will already be set on the host component :thinking:

Comment (Kiarokh) [reply src/components/switch/switch.tsx:124]: Oh… I didn't know. tnx

Comment (Kiarokh) [inline src/components/switch/switch.tsx:124]:
```suggestion
            <Host>
```

Comment (Kiarokh) [reply src/components/switch/switch.tsx]: ⚡

Comment (jgroth) [reply src/components/switch/switch.tsx]: Clicking the "invalid" checkbox in the switch example does not seem to do anything. 
Is it due to this? Maybe it's needed anyway? I don't see how though :smile:

Comment (Kiarokh) [reply src/components/switch/switch.tsx]: I only need to the `invalid` tag in the HTML:
```html
<limel-switch invalid></limel-switch>
```
If there is a helperText, then the helper text turns red. This will happen in [this PR](https://github.com/Lundalogik/lime-elements/pull/2591) as soon as it is merged.

However, we might also want to make the label of the Switch red. I can do that later in a follow up PR.

**Review by jgroth — APPROVED** [review]
